tests/tests_twobody/test_orbit.py

```diff
@@ -760,6 +760,28 @@ def test_heliosynchronous_orbit_a():
     assert_quantity_allclose(ss0.ecc, expected_ecc)
 
 
+def test_heliosynchronous_orbit_ecc():
+    # Vallado, example 11-2b
+    expected_ecc = 0.0 * u.one
+    expected_inc = 98.6 * u.deg
+    expected_a = 7178.1363 * u.km
+    ss0 = Orbit.heliosynchronous(Earth, a=expected_a, inc=expected_inc)
+
+    assert_quantity_allclose(ss0.inc, expected_inc, rtol=1e-4)
+    assert_quantity_allclose(ss0.a, expected_a, rtol=1e-5)
+    assert_quantity_allclose(ss0.ecc, expected_ecc)
```

Comment:
If I understand correctly, this corresponds to example 11-2a, right? I'm now trying to reproduce the results and I don't get the same. Am I doing something wrong?

```
>>> math.degrees(math.acos(-((1.12532 ** (7/2) * 2 * 1.991063853e-7) / (3 * 6378.137**2 * 0.00108263 * math.sqrt(398600.4418)))))
90.00000000000043
```

Reply:
It seems the equations in the solution of example 11-2:

![Screenshot from 2021-07-09 19-47-34](https://user-images.githubusercontent.com/68844397/125092095-987b7c80-e0ee-11eb-97bb-0089da6dc98a.png)

The value of `a` is printed as 1.12532 where it must have been 7178.1363. If I use 7178.1363 instead, in the equation, it gives me an inclination of 98.60308111784582. So maybe that's not a problem?

Reply:
Thanks for reproducing it! Yeah, shouldn't be a problem